tardis/io/model/parse_density_configuration.py

```diff
@@ -93,12 +91,14 @@ def parse_config_v1_density(config: Configuration) -> u.Quantity:
     v_middle = adjusted_velocity[1:] * 0.5 + adjusted_velocity[:-1] * 0.5
```

Comment:
Maybe we should think about this adjusted velocity?

Reply:
Probably! This comes back to finite volume